notebooks/official/feature_store/vertex_ai_feature_store_based_llm_grounding_tutorial.ipynb

```diff
@@ -0,0 +1,1255 @@
+{
```

Comment:
<p>Add the links to Colab, Workbench and Github. See example at <a href="https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb" target="_blank">https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2652/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Done</p>